tests/baselines/reference/ts-expect-error.errors.txt

```diff
@@ -67,4 +67,18 @@ ts-expect-error.ts(40,2): error TS2367: This comparison appears to be unintentio
 !!! error TS2367: This comparison appears to be unintentional because the types 'true' and 'false' have no overlap.
     (({ a: true } as const).a === false); // error
      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-!!! error TS2367: This comparison appears to be unintentional because the types 'true' and 'false' have no overlap.
\ No newline at end of file
+!!! error TS2367: This comparison appears to be unintentional because the types 'true' and 'false' have no overlap.
+    
+    // @ts-expect-error: additional commenting with no whitespace
+    var invalidCommentedFancySingle: number = 'nope';
+    
+    /*
+     @ts-expect-error: additional commenting with no whitespace */
+    var invalidCommentedFancyMulti: number = 'nope';
+    
+    // @ts-expect-error--additional commenting with no whitespace
+    var invalidCommentedFancySingle: number = 'nope';
+    
+    /*
+     @ts-expect-error--additional commenting with no whitespace */

```

Comment:
In any case, I don't think it's necessary to skip double dashes. Just an immediately trailing colon.